src/backend/Dockerfile.lineage_qc

```diff
@@ -0,0 +1,40 @@
+FROM python:3.9.1-slim AS base
```

Comment:
Can you use python:3.10-slim instead? I've been meaning to upgrade the other images now that nextstrain is on a newer python, but haven't had a chance. At least if we build this one on the newer version we won't have to worry about fixing it when we upgrade.

Reply:
oh sure!

Reply:
hmm that seems to break the poetry install..

Reply:
we might have to wait on upgrading for now until we have more time to debug if that's ok, maybe it can be a follow up PR